dbms/src/TestUtils/mockExecutor.h

```diff
@@ -145,6 +147,9 @@ class DAGRequestBuilder
     DAGRequestBuilder & sort(MockOrderByItem order_by, bool is_partial_sort, uint64_t fine_grained_shuffle_stream_count = 0);
     DAGRequestBuilder & sort(MockOrderByItemVec order_by_vec, bool is_partial_sort, uint64_t fine_grained_shuffle_stream_count = 0);
 
+    // expand
+    DAGRequestBuilder & expand(MockVVecColumnNameVec grouping_set_columns);
```

Comment:
please add unit test for mock expand in https://github.com/pingcap/tiflash/blob/master/dbms/src/TestUtils/tests/gtest_mock_executors.cpp#L302-L320